tests/sql_mock/clickhouse/test_column_mocks.py

```diff
@@ -1,9 +1,9 @@
-from sql_mock.clickhouse.column_mocks import Array, ClickhouseColumnMock, Decimal
+from sql_mock.clickhouse.column_mocks import Array, ClickhouseColumnMock, Decimal, String, Int
 
 
 def test_init_not_nullable():
     """
-    ...then nullable should be False and dtype be the same as passed.
+    ...then nullable should be True and dtype be the same as passed.
```

Comment:
**nitpick:** I think we need to change this back. Also below it seems that it should stay as is